spec/fixtures/sample_objects/NMGRP0001.txt

```diff
@@ -0,0 +1 @@
0\" attached_pic=\"0\"/>\n            <tag key=\"language\" value=\"eng\"/>\n            <tag key=\"handler_name\" value=\"DataHandler\"/>\n        </stream>\n    </streams>\n\n    <format filename=\"/srv/scratch/transcoder_workspace_xcode-04_dev/NONMDPI0000888.downloading/data/NONMDPI0000888_01_mezz.mov\" nb_streams=\"5\" nb_programs=\"0\" format_name=\"mov,mp4,m4a,3gp,3g2,mj2\" format_long_name=\"QuickTime / MOV\" start_time=\"0.000000\" duration=\"354.788000\" size=\"2422702631\" bit_rate=\"54628738\" probe_score=\"100\">\n        <tag key=\"major_brand\" value=\"qt  \"/>\n        <tag key=\"minor_version\" value=\"512\"/>\n        <tag key=\"compatible_brands\" value=\"qt  \"/>\n        <tag key=\"title\" value=\"Wilderness Plots: Songs and Stories of the Prairie Promo. Master - SD Spot (Exchange)\"/>\n        <tag key=\"encoder\" value=\"Lavf57.0.100\"/>\n        <tag key=\"comment\" value=\"File Created by Memnon Archiving Services\"/>\n    </format>\n</ffprobe>\n","filename":"NONMDPI0000888_01_med.mp4","url_rtmp":"rtmp://bl-uits-ct-mdpi.uits.indiana.edu:1935/avalon_dark/_definst_/mp4:B-RTVS/NMGRP0001_NONMDPI0000888_01_med_20160108_093019.mp4"}},"master_md5":"bc5bd4f942e55affbe29b643c58fded0"}},"part":3}],"metadata":{"audio":"false","format":{"0000888":"audio comment"},"unit":"B-RTVS","mods":"<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n<mods xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\" xmlns=\"http://www.loc.gov/mods/v3\" version=\"3.5\" xsi:schemaLocation=\"http://www.loc.gov/mods/v3 http://www.loc.gov/standards/mods/v3/mods-3-5.xsd\">\n  <titleInfo>\n    <title>Wilderness Plots: Songs and Stories of the Prairie Promo. Master - SD Spot (Exchange)</title>\n  </titleInfo>\n  <name type=\"personal\" usage=\"primary\">\n    <namePart/>\n  </name>\n  <originInfo>\n    <dateIssued/>\n  </originInfo>\n  <typeOfResource>moving image</typeOfResource>\n  <physicalDescription>\n    <form authority=\"gmd\">videorecording</form>\n    <form authority=\"marcsmd\">videocassette</form>\n  </physicalDescription>\n  <note displayLabel=\"system details\">Betacam</note>\n  <identifier type=\"local\" displayLabel=\"Unit\">B-RTVS</identifier>\n  <identifier type=\"local\" displayLabel=\"Call Number\"/>\n  <identifier type=\"local\" displayLabel=\"Collection Identifier\"/>\n  <identifier type=\"local\" displayLabel=\"MDPI Barcode\">0000888</identifier>\n  <recordInfo>\n    <recordCreationDate encoding=\"iso8601\">2015-09-17T19:45:25-04:00</recordCreationDate>\n    <recordChangeDate encoding=\"iso8601\">2015-11-05T20:48:13-05:00</recordChangeDate>\n    <recordIdentifier source=\"MDPI\">0000888</recordIdentifier>\n  </recordInfo>\n<name type=\"personal\"><namePart>Kim, Joo.</namePart><role><roleTerm authority=\"marcrelator\" type=\"code\">ctb</roleTerm><roleTerm type=\"text\">Contributor</roleTerm></role></name></mods>\n","mods_source":"POD"},"comments":[["NONMDPI Object 0000888","description of object"],["NONMDPI0000888_01_low.mp4","Ingest notes ..."]]}
```

Comment:
I edited this to remove the streaming server URIs for IU and commited to this branch. Also added formatting and indenting.